.github/workflows/daily-build-check.yml

```
@@ -0,0 +1,96 @@
+          echo "Latest commit on auth_rbac: $LATEST_COMMIT"
+          echo "LATEST_COMMIT=$LATEST_COMMIT" >> $GITHUB_ENV
+
+      - name: Compare commits and trigger build if different
+        if: env.LATEST_COMMIT != env.LAST_SUCCESSFUL_COMMIT
+        run: |
+          echo "New commits found. Proceeding with build and deployment."
+        
+      # The following steps will only be executed if the previous condition is true
+      - name: Set up Docker Buildx
+        if: env.LATEST_COMMIT != env.LAST_SUCCESSFUL_COMMIT
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)